src/templates/file.html

```diff
@@ -0,0 +1,38 @@
+{% import "macros.html" as macros %}
```

Comment:
I ran that stuff on grcov itself and open for example `parser.rs.html` and there are a lot of useless `\n` or `<space>`.
It'd be nice to strip them to reduce the size of generated files.